src/Console/OutputFormatter/DotFormatter.php

```diff
@@ -66,7 +66,8 @@ public function start(int $mutationCount): void
             . '<escaped>M</escaped>: escaped, '
             . '<uncovered>S</uncovered>: uncovered, '
             . '<with-error>E</with-error>: fatal error, '
-            . '<timeout>T</timeout>: timed out',
+            . '<timeout>T</timeout>: timed out, '
+            . '<constrained>C</constrained>: constrained',
```

Comment:
I admit I'm not sold on the term "constrained". I feel like having "Skipped" with an explanation as to why (e.g.: not tested, timeout exceed limit, etc.) would be both more expressive and clearer: the user would not have to wonder _what_ "constrained" means, but rather wonder _why_ the mutation was skipped.

This also fits better for future changes where say we skip mutations through another optimization strategy. The typical example being the top-down strategy: you skip the nested mutations if the top one escaped. When implementing it, we would only have to use the existing "skipped" and provide another reason. Whereas if we go with "constrainted", it's really unclear what we should do there

Reply:
OK, I don't mind "skipped", but we used `S` for uncovered. How do you propose we tag these skipped in the progress log?

Reply:
`U` for uncovered? (should be changed in a different PR though)

Reply:
OK, prints `S: skipped` now